cmd/webhook/main.go

```diff
@@ -132,7 +132,7 @@ func newValidationAdmissionController(ctx context.Context, cmw configmap.Watcher
 	)
 }
 
-func newConfigValidationController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+func newConfigValidationController(ctx context.Context, _ configmap.Watcher) *controller.Impl {
```

Comment:
As this is only Serving visible, you can also just remove the param completely

Reply:
The above func is being injected in "WebhookMainWithContext" at line 170 which is of type - "ControllerConstructor func(context.Context, configmap.Watcher) *controller.Impl".
If I remove the param, it is breaking this dependency injection.